lab-justin/gulpfile.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+gulp.task('default', function(){
```

Comment:
Instead of using the default task for your test task I suggest creating an independent test task like you've done with 'eslint' and instead using your default task to run both the eslint and test tasks.